src/pages/home/report/ReportActionItemSingle.js

```diff
@@ -74,7 +74,7 @@ function ReportActionItemSingle(props) {
     const actorAccountID = props.action.actorAccountID;
     let {displayName} = props.personalDetailsList[actorAccountID] || {};
     const {avatar, pendingFields} = props.personalDetailsList[actorAccountID] || {};
-    let actorHint = lodashGet(props.action, 'actorEmail', '').replace(CONST.REGEX.MERGED_ACCOUNT_PREFIX, '');
+    let actorHint = displayName.replace(CONST.REGEX.MERGED_ACCOUNT_PREFIX, '');
```

Comment:
Should we get email here from personal details list and if email is not present then fallback to login?
I think here we are trying to remove prefix from merged accounts emails

Reply:
Juuuust to be clear @MonilBhavsar do you mean, we should check the `login` from `personalDetailsList` first - and if that doesn't exist, we fall back to `displayName`?

If that's what you meant, then yes I agree :D

Reply:
> do you mean, we should check the login from personalDetailsList first - and if that doesn't exist, we fall back to displayName?

Yes correct! Exactly what I mean :D

> displayName from personalDetailsList always fallback to login if display name is not set, isn't it?

Yes, in case display name is set it won't be login. And in that line we specifically want to process login(email). So we can first check if login exists, if not we can fallback to displayName.